packages/generator-clay-component/app/templates/src/tests/_Boilerplate.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * © 2018 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import * as React from 'react';
+import * as TestRenderer from 'react-test-renderer';
+import <%= componentName %> from '../<%= componentName %>';
+
+describe('<%= componentName %>', function() {
```

Comment:
```suggestion
describe('<%= componentName %>', () => {
```
I see you're already using arrow functions in the `it` examples.